project/Dependencies.scala

```diff
@@ -16,6 +16,8 @@ object Dependencies {
 
   val scalafmt = "org.scalameta" %% "scalafmt-dynamic" % "3.7.17"
 
+  val `scala-logging` = "com.typesafe.scala-logging" %% "scala-logging"   % "3.9.5"
```

Comment:
We should not depend on this library, ZIO has it's own built-in logging API

Reply:
Used built-in logging